content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -69,31 +69,27 @@ data:
 EOF
 {{< /text >}}
 
-### Install Via Helm
+### Install Via `Istioctl`
 
 Once you create the Kiali secret, follow
```

Comment:
What about something like
-- `Once you create the Kiali secret`
++ `You are now ready to install Kiali.`

Reply:
I am hesitant to alter sentence structures/verbiage for parts in this document that are unrelated to the changes required, unless it is necessary. It took **a lot** of back-and-forth with the Istio doc writers when we originally wrote this page. I left everything as it was unless I needed to explicitly change something.

Reply:
ok, sounds good, we can improve this later in a follow up.